ocp_utilities/exceptions.py

```diff
@@ -6,6 +6,14 @@ class NodeUnschedulableError(Exception):
     pass
 
 
+class PodsHealthCheckError(Exception):
```

Comment:
Were this exception used?

Reply:
Its not used this was intended to be an alternate exception. Will remove.